src/System.CommandLine/src/System/CommandLine/ArgumentSyntax.cs

```diff
@@ -52,12 +53,7 @@ private void Validate()
                 var helpText = GetHelpText();
                 Console.Error.Write(helpText);
 
-                // TODO: This should use Environment.Exit(0) but this API isn't available yet.
```

Comment:
With this change, how will the caller of Parse know that the arguments passed where asking for help and that the execution of the program should stop?